util/after-build.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+trap 'echo -e "\033[1m\033[31m==> ERROR:\033[0m \033[1mCommand failed\033[0m" >&2' ERR
+set -eE
+
+msg() {
+  echo -e "\033[1m\033[32m==>\033[0m \033[1m$1\033[0m"
+}
+
+package_linux() {
+  make install INSTALL_ROOT="${PWD}/Pencil2D"
+
+  msg "Creating AppImage..."
+  # "Downgrade" the desktop entry to version 1.0
+  sed -i "/^Keywords\(\[[a-zA-Z_.@]\+\]\)\?=/d;/^Version=/cVersion=1.0" \
+    Pencil2D/usr/share/applications/pencil2d.desktop
+  install -Dm755 /usr/bin/ffmpeg Pencil2D/usr/plugins/ffmpeg
+  install -Dm755 "/usr/lib/x86_64-linux-gnu/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner" \
+    "Pencil2D/usr/lib/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner"
+  local gst_executables="-executable=Pencil2D/usr/lib/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner"
+  for plugin in adpcmdec alsa app audioconvert audioparsers audioresample \
+      autodetect coreelements gsm id3demux jack mpg123 mulaw playback \
+      pulse typefindfunctions wavparse; do
```

Comment:
There are some mp3 files that I am still not able to play. I believe this is because they contain APE tags (https://en.wikipedia.org/wiki/APE_tag). I think if we include the `apetag` gstreamer plugin it should fix this. However I'm not confident that this plugin exists in this version of Ubuntu as it is using as some of my plugins seem have different names on 20.04.